File: sharedlife-frontend/src/app/models/taskTraking/taskTrakingDTO.ts
export class TaskTrakingDTO{
    id: string;
    creationDate: string;
    startDate: string;
    endDate: string;
    constructor(i?: string, cd?: string, sd?: string, ed?:string){
        this.id = i ?? "";
        this.creationDate = cd ?? "";
        this.startDate = sd ?? "";
        this.endDate = ed ?? "";
    }

}

File: sharedlife-backend/src/main/java/net/tfg/sharedlife/dto/TaskTrakingDTO.java
package net.tfg.sharedlife.dto;

import java.util.Date;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
@ToString
@Getter
@Setter
public class TaskTrakingDTO {
    private Long id;
    private Date creationDate;
    private Date startDate;
    private Date endDate;
}
